Dockerfile

```diff
@@ -1,20 +0,0 @@
-FROM ruby:2.6.6
-RUN gem install foreman
-
-ENV RABBITMQ_HOSTS rabbitmq
-ENV RABBITMQ_VHOST /
-ENV RABBITMQ_USER guest
-ENV RABBITMQ_PASSWORD guest
-ENV RABBITMQ_EXCHANGE published_documents
-ENV RAILS_ENV development
-ENV REDIS_HOST redis
-
-ENV APP_HOME /app
-RUN mkdir $APP_HOME
-
-WORKDIR $APP_HOME
-ADD Gemfile* $APP_HOME/
-RUN bundle install
-ADD . $APP_HOME
-
-CMD foreman run major-change-queue-consumer
```

Comment:
I can't see this one project causing problems but if we were to start deleting a few of these we'd want to have a chat with replatforming as they are using these Dockerfiles to run apps in fargate. Removing them alone would mean we get left with an image on docker-hub but it no longer updated.

Reply:
Yes, indeed. I think they're already having some issues with these images, and I don't think they're using this one, since they're focussing on frontend apps at the moment.

(I'm not planning on deleting any more of them.)